src/yoroi-wallets/cardano/byron/ByronWallet.ts

```diff
@@ -993,19 +988,19 @@ export class CardanoWallet implements YoroiWallet {
   }
 
```

Comment:
there are many methods above that on `ByronWallet` that should also throw (create txs -> staking key relations/withdrawal tx/delegation / etc...), also there is some `isByron` I think it should be dropped

Reply:
this is all the exact behavior Byron wallets currently have. I want to change as little as possible